itest/assertions.go

```diff
@@ -265,14 +265,42 @@ func verifyProofBlob(t *testing.T, tapd *tapdHarness,
 	anchorTxBlockHeight := rpcAsset.ChainAnchor.BlockHeight
 	require.Greater(t, anchorTxBlockHeight, uint32(0))
 
-	headerVerifier := func(blockHeader wire.BlockHeader) error {
-		hash := blockHeader.BlockHash()
+	headerVerifier := func(header wire.BlockHeader, height uint32) error {
+		hash := header.BlockHash()
 		req := &chainrpc.GetBlockRequest{
 			BlockHash: hash.CloneBytes(),
 		}
 		_, err := tapd.cfg.LndNode.RPC.ChainKit.GetBlock(ctxb, req)
-		return err
+		if err != nil {
```

Comment:
Same here, probably makes sense to only do this after we've confirmed the block should be in the current chain.